documentation/overview_and_index.md

```diff
@@ -0,0 +1,54 @@
+## Problem Statement
+[Human Rights First (HRF)](https://www.humanrightsfirst.org/)   is an independent advocacy and 
+action organization that challenges America to live up 
+to its ideals. HRF is seeking a data search and 
+visualization tool to provide the public with a 
+user-friendly view into their large data set of asylum 
+case decisions. Similar to [the TRAC database](https://trac.syr.edu/phptools/immigration/asylum/)—
+a popular existing tool, but with many limitations.
+
+This design document describes the HRF Asylum Report
+Generator backend, a new service that will allow users
+to view and filter decision data on multiple parameters,
+generate visualizations and download search results.
```

Comment:
Oxford comma here, and perhaps rewording:
"This design document describes the backend of the HRF Asylum Report Generator, a new service that will allow ... generate visualizations, and download search results."

Reply:
Good catch!